paymentForm.html

```diff
@@ -0,0 +1,42 @@
+<html lang="en">
+<head>
+</head>
+
+<body>
+    <form action="" method="get">
+
+    <h1>Payment Form</h1>
+    <p>Require fields are makred by <u>*</u> </p>
+    <h3>Contact Information</h3>
+    <ul>
+         <input type="radio">Mr. <br>
+         <input type="radio">Ms. <br>
+         <input type="radio">Mrs. <br>
+         <input type="radio">Mx. <br>
+         <input type="radio">Cat <br>
+
+    </ul>
+    Name:* <input type="text"><br>
+    Email:* <input type="text"><br>
+    <h3>Payment Information</h3>
+    Card Type <select>
```

Comment:
Try to keep both beginning tag and ending tags on the same indentation line, makes it easier to read